Solutions/Rapid7InsightVM/Playbooks/Playbooks/Rapid7InsightVM-EnrichVulnerabilityInfo/readme.md

```diff
@@ -0,0 +1,44 @@
+# Rapid7InsightVM-EnrichVulnerabilityInfo
+
+## Summary
+
+When a new sentinel incident is created, this playbook gets triggered and performs the following actions:
+
+1. Obtains IPs from the incident.
+2. Searches asset ids by the IPs.
+3. Gets vulnerability ids.
+4. Gets vulnerability information.
+5. Adds obtained information as a comment to the incident.
+
+<img src="./playbook_screenshot.png" width="50%"/><br>
```

Comment:
Update the screenshot with the latest copy. Action names in the image do not match with what gets deployed.